docs/weblog/README.md

```diff
@@ -734,6 +734,12 @@ This endpoint get a `name` and a `value` form the query string, and adds a heade
 
 This endpoint is the initial endpoint used to test session fingerprints, consequently it must initialize a new session and the web client should be able to deal with the persistence mechanism (e.g. cookies).
 
+Returns the identifier of the created session id :
+
+```text
+c377db41-b664-4e30-af57-5df2e803bec7
+```
```

Comment:
Otherwise it looks like you want us to return an hardcoded value
```suggestion
Returns the identifier of the created session id. Example :

```text
c377db41-b664-4e30-af57-5df2e803bec7
```
```